ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -218,19 +221,20 @@ def write_netCDF_aggfile(agg_dataset, ncout_filename, encoding):
     :return: name of the netCDf file written
     """
 
-    agg_dataset.to_netcdf(ncout_filename, encoding=encoding, format='NETCDF4_CLASSIC')
+    agg_dataset.to_netcdf(base_path + ncout_filename, encoding=encoding, format='NETCDF4_CLASSIC')
```

Comment:
You should use `os.path.join(base_path, ncout_filename)` to create the output file path.

Reply:
replaced.